### DISCUSSION TITLE
form.validateFields 返回 true ？

### DISCUSSION THREAD

**Discussion by xingxinglieo** [ant-design/ant-design#35997] [Q&A]
form.validateFields 预期返回整个表单，不知道什么情况下会返回 true，getFieldsValue 获取正常，现在还没有定位到。。。但是按照官方稳定只要是通过校验都应该返回整个表单。想问下有遇到同样情况的吗？什么情况下会返回 true？

**Comment by xingxinglieo** [comment]
这个问题刚刚发现的，给业务埋下巨大隐患。。。

**Reply by yoyo837** [reply]
没遇到过，上在线代码示例
